hikari/permissions.py

```diff
@@ -198,3 +254,29 @@ class Permissions(enums.Flag):
         development" by Discord meaning that "it may be changed or removed"
         without warning.
     """
+
+    MANAGE_THREADS = 1 << 34
+    """Allows for deleting and archiving threads, and viewing all private threads.
+
+     !!! note
+        In guilds with server-wide 2FA enabled this permission can only be used
+        by users who have two-factor authentication enabled on their account
+        (or their owner's account in the case of bot users) and the guild owner.
+    """
+
+    USE_PUBLIC_THREADS = 1 << 35
+    """Allows for creating and participating in threads."""
+
+    USE_PRIVATE_THREADS = 1 << 36
+    """Allows for creating and participating in private threads."""
+
+    @classmethod
+    def all_permissions(cls) -> Permissions:
+        """Get an instance of `Permissions` with all the known permissions.
+
+        Returns
+        -------
+        Permissions
```

Comment:
```suggestion
        hikari.permissions.Permissions
```

Reply:
why does that need to be an absolute link when it's in the current module? Seems to generate fine on my machine 
![image](https://user-images.githubusercontent.com/11181703/121196306-4550a880-c868-11eb-9281-68db60bd61e2.png)